Editor/Editor.vb

```diff
@@ -65,7 +66,11 @@ Public Class Editor
                             Dim path As String = System.IO.Path.GetDirectoryName(m_filename)
                             Dim filter As String = System.IO.Path.GetFileName(m_filename)
                             m_fileWatcher = New System.IO.FileSystemWatcher(path, "*.aslx")
+                            m_fileWatcher.IncludeSubdirectories = True
                             m_fileWatcher.EnableRaisingEvents = True
+                            m_fileWatcherQuest = New System.IO.FileSystemWatcher(Application.StartupPath + "\Core", "*.aslx")

```

Comment:
I'm not entirely convinced of this change, I don't think anyone should be editing core library `.aslx` files directly within the Quest app directory itself, while the editor is loaded?

Reply:
This was all @Pertex. (Not sure if he gets notified without tagging him here.)

We never really discussed this one; so, I don't know what his reason was. I'm sure he'll be along soon, though.

Reply:
I have added these things especially for quest developers, when you need to quickly edit core files. An addition, but not important for the normal user.